python/chemiscope/input.py

```diff
@@ -376,8 +380,10 @@ def create_input(
     # Check to tell the user they might have forgotten some properties coming
     # from the frames (that chemiscope used to automatically extract). This code
     # should be removed in version 0.6 of chemiscope.
-    if frames is not None:
+    # stk cannot have atom properties or structure properties, so skipping.
```

Comment:
could you instead implement the `_list_atom_properties`/`_list_structure_properties` but make then return an empty list?  This way the code in this file does not have to know about all possible data sources

Reply:
I have done this. Makes sense.